client/state/current-user/reducer.js

```diff
@@ -98,6 +110,7 @@ export function capabilities( state = {}, action ) {
 
 export default combineReducers( {
 	id,
+	geoLocation,
```

Comment:
It makes sense to have the geo location attached to `current-user` in Calypso, but the way I understand it it's currently very much coupled to Happychat.

For this reason I think we should move this under the `happychat` branch of the Redux store, since the only way this can be populated is by starting a happychat session. An alternative would be to decouple the geo fetching from Happychat entirely, perhaps with a new wpcom endpoint, but I think that's probably not worth the cost to get this feature going.